src/ripple/protocol/TxFlags.h

```diff
@@ -150,6 +150,10 @@ constexpr std::uint32_t const tfNFTokenCancelOfferMask = ~(tfUniversal);
 // NFTokenAcceptOffer flags:
 constexpr std::uint32_t const tfNFTokenAcceptOfferMask = ~tfUniversal;
 
+// BridgeModify flags:
+constexpr std::uint32_t tfClearAccountCreateAmount     = 0x00010000;
+constexpr std::uint32_t tfBridgeModiryMask = ~(tfUniversal | tfClearAccountCreateAmount);
```

Comment:
typo: should be `tfBridgeModifyMask`

Reply:
Fixed in 2bcd5f84a5 [fold] Fix modify mask mispelling: https://github.com/XRPLF/rippled/pull/4292#pullrequestreview-1298521511